boards/renesas/ek_ra8d1/Kconfig.defconfig

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 32768 if ULPT_RENESAS_RA
+	default 480000000
```

Comment:
why is this in board config? This is a SoC config item, it should be in soc/*/Kconfig.defconfig

Reply:
@nordicjm . I moved it in the zephyr/soc/renesas/ra/ra8*/Kconfig.defconfig